docs/Overview.md

```diff
@@ -0,0 +1,79 @@
+# Da Vinci PAS Test Kit Overview
+
+This document provides a high-level overview of the Da Vinci Prior Authorization Support (PAS) Test Kit, its purpose, and general testing approach.
+
+## Purpose
+
+The Da Vinci PAS Test Kit is designed to validate the conformance of healthcare IT systems against [version 2.0.1 of the HL7 FHIR Da Vinci Prior Authorization Support (PAS) Implementation Guide (IG)](https://hl7.org/fhir/us/davinci-pas/STU2/). It helps implementers ensure their systems can correctly participate in electronic prior authorization workflows as defined by the PAS IG.
+
+The test kit is built using the [Inferno Framework](https://inferno-framework.github.io/), an open-source platform for building FHIR-based test kits.
+
+## Test Kit Structure
+
+The PAS Test Kit includes two main test suites:
+
+* **Server Test Suite**: For systems acting as payers (see [Server Details](Server-Details.md) for more information)
+* **Client Test Suite**: For systems acting as providers (see [Client Details](Client-Details.md) for more information)
+
+## General Testing Approach
+
+The test kit validates systems through:
+
+1. **Workflow Simulation**: Tests guide the system through key PAS workflows including:
+   * Prior authorization request submission and response handling
+   * Approval, denial, and pended decision flows
+   * Error condition handling
+
+2. **Data Conformance**:
+   * Validation of must-support elements in PAS-defined FHIR profiles
+   * FHIR resource validation using the official FHIR validator
+   * Verification of proper Bundle structure and references
+
+3. **Authentication**:
+   * Support for SMART Backend Services
+   * UDAP B2B client credentials flow
+   * Other authentication methods via attestation
```

Comment:
These lists don't really follow a parallel structure, making it feel disjointed and awkward.  Not sure that it is worth reworking, but wanted to mention my hestitation about it.

Reply:
Yeah, I was a bit hesitant about this rephrasing too.  I'll take another pass.  I think, the idea here is that at the 'Test Kit Overview' level I think we should state the general testing approach / philosophy.  Then in the suite details talk specifics.

Reply:
We can circle back on this after it is all in. If it feels awkward we can update it.